src/PrestaShopBundle/Kernel/ModuleRepository.php

```diff
@@ -55,8 +56,14 @@ final class ModuleRepository
      */
     private $activeModulesPaths;
 
-    public function __construct(Connection $connection, $databasePrefix)
+    /**
+     * @var Context
+     */
+    private $context;
+
+    public function __construct(Connection $connection, $databasePrefix, LegacyContext $context)
```

Comment:
Hi, adding this 3rd argument changes the constructor. If we make it optional, it does not break any external class using it 😊  .

Reply:
done 😊